symbols/overlay29.yml

```diff
@@ -241,15 +241,16 @@ overlay29:
         r0: string ID (unclear what this means)
         r1: entity pointer
         r2: ?
-    - name: TryActivateMapSurveyorNA
+    - name: UpdateMapSurveyorFlag
       address:
         NA: 0x22E2DD8
+        EU: 0x22E375C
       description: |-
         Sets the Map Surveyor flag in the dungeon struct to true if a team member has Map Surveyor, sets it to false otherwise.
         
-        Exclusive to the NA version; the EU equivalent returns true if the flag was changed.
+        This function has two variants: in the EU ROM, it will return true if the flag was changed. The NA version always returns 0x41 (equivalent to true).
```

Comment:
> The NA version always returns 0x41 (equivalent to true)
This is incorrect, this function calls `TeamMemberHasEnabledIqSkill` after the `mov r0,41h`, so it returns the return value of `TeamMemberHasEnabledIqSkill` (so whether a team member has map surveyor enabled in this case).

Reply:
Oh that's right, then what about:
```
[...] The NA version will return the new value of the flag.
```
(since r0 gets stored as the new flag)

Reply:
Sounds good

Reply:
Oops this was my suggestion, apparently I don't know how to read assembly 🙃